Documentation/docs/releases/4.0.md

```diff
@@ -150,53 +150,53 @@ More details here:
 ### Numerical Libraries
 
 -   [Refactor Numerical
-    Libraries](https://itk.org/Wiki/ITK_Release_4/Refactor_Numerical_Libraries)
+    Libraries](https://insightsoftwareconsortium.github.io/ITKWikiArchive/Wiki/ITK_Release_4/Refactor_Numerical_Libraries)
```

Comment:
Now broken, it had non-trivial content.

Reply:
@thewtex There are a few more unaddressed review comments (such as this one). You have to click on "6 hidden conversations Load more..." to see them.

Reply:
```suggestion
    Libraries](https://insightsoftwareconsortium.github.io/ITKWikiArchive/Wiki/ITK_Release_4/Refactor_Numerical_Libraries/Refactor_Numerical_Libraries/)
```